### ISSUE TITLE
Zero plugin depedencies leads to warning

### ISSUE THREAD

**Issue by dnaber-de** [wecodemore/wpstarter#62]
If I setup a new installation without having any plugin defined as dependency, I get the following warning:

```
Warning: opendir(/var/www/public/wp-content/plugins): failed to open dir: No such file or directory in /var/www/public/wp/wp-admin/includes/plugin.php on line 259
```

The problem is, that there's no plugin directory, of course. Not sure how this can be handled. Maybe a post install script checking for the existence of this directory and creates it if necessary.

**Comment by franz-josef-kaiser** [comment]
Will take a look at this tomorrow.

**Comment by GaryJones** [comment]
https://youtu.be/Yop62wQH498?t=56s

> Tomorrow, tomorrow
> I love ya, tomorrow,
> You're only 5 months away.

**Comment by gmazzap** [comment]
Only after 1 e 6 months later month I am on this :D

So it seems that WordPress does a `@opendir` so no error should happen , but I guess you found that warning in logs.

I think it makes sense to do the same that we do for themes, that is, create an empty folder in case Composer did not create a folder because there was no plugin dependency. 

Even if for themes is more important because the frontoffice is completely missing otherwise (see https://core.trac.wordpress.org/ticket/31620) avoiding a warning is always a good thing.

I will do this for both v3 and v2.

**Comment by gmazzap** [comment]
For v2 this has been included in [2.4.4](https://github.com/wecodemore/wpstarter/releases/tag/2.4.4) for v3 it is also included.

We can close this. Thanks to anyone involved.
